service/src/integration-test/java/uk/nhs/adaptors/oneoneone/cda/report/controller/ReportControllerIT.java

```diff
@@ -63,6 +68,7 @@
         + ":SimpleMessageResponse>";
     private static final List<String> IGNORED_JSON_PATHS = List.of(
         "entry[*].fullUrl",
+        "entry[*].resource.address[*]",
```

Comment:
we don't want to ignore address -  we want to modify the expected json message to reflect the changes you made